by-language/csharp-npgsql/demo.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Npgsql" Version="7.0.6" />
+    <PackageReference Include="Npgsql" Version="8.0.0" />
```

Comment:
Those tests employ a test matrix, where versions are defined elsewhere.

-- https://github.com/crate/cratedb-examples/blob/06c04f6a3/.github/workflows/test-npgsql.yml#L46

Reply:
GH-169 is the right choice to update the Npgsql version.